scripts/__commons.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env sh
```

Comment:
I'd recommend removing this line. Use:

# shellcheck shell=bash

instead. This makes it clear that this file is not an executable but a library instead.

Reply:
Awesome. This is super!